README.md

```diff
@@ -28,19 +30,55 @@ That inspired me to create my own. I started using images only and eventually sw
 One of the main things that killed me during developing this was color detection. It works for my
 room, but I bet it doesn't work for you, or you must have the same lighting and color scheme as I do.
 
+# Prerequisites (relevant only when using pipenv)
+You will need to install `pipenv` on your machine as well as have python 3.7 installed.
+#### Installing pipenv: ([Full Guide](https://pypi.org/project/pipenv/))
+```
+$ sudo apt install pipenv
+```
+
+NOTE: If you have (and need also) an earlier than 3.7 version of python installed, you might need to used [pyenv](https://github.com/pyenv/pyenv-installer) to manage multiple python versions on your machine.
+
 # Installation
-Start off by cloning:
+###Start off by cloning:
```

Comment:
Add a whitespace in between `###` and the word `Start` and add a new empty line before and after every heading. Please do this for all headings.

Reply:
Fixed (in the base review - https://github.com/kkoomen/qbr/pull/6)!